features/note_tags.feature

```diff
@@ -0,0 +1,17 @@
+Feature: User tags notes
+
+  Scenario: User creates a note
+    Given I have an existing user account
```

Comment:
I can show you how to shorten this and reduce repetition in later tests. It looks good otherwise.